tirex.robot

```
@@ -0,0 +1,599 @@
+  ${tender_data}=   adapt_procuringEntity   ${role_name}   ${tender_data}
+  [return]  ${tender_data}
+
+Підготувати клієнт для користувача
+  [Arguments]  ${username}
+  Open Browser  ${USERS.users['${username}'].homepage}  ${USERS.users['${username}'].browser}  alias=${username}
+  Set Window Size  @{USERS.users['${username}'].size}
+  Set Window Position  @{USERS.users['${username}'].position}
+  Run Keyword If  '${username}' != 'tirex_Viewer_auction'  Run Keywords
+  ...  Login  ${username}
+  ...  AND  Run Keyword And Ignore Error  Wait Until Keyword Succeeds  10 x  1 s  Закрити модалку з новинами
+
+Закрити модалку з новинами
+  Wait Until Element Is Enabled   xpath=//button[@data-dismiss="modal"]
+  Click Element   xpath=//button[@data-dismiss="modal"]
+  Wait Until Element Is Not Visible  xpath=//button[@data-dismiss="modal"]
+
+Login
+  [Arguments]  ${username}
+  Click Element  xpath=//a[@href="/login"]
+  Wait Until Page Contains Element  id=loginform-username  10
+  Input text  id=loginform-username  ${USERS.users['${username}'].login}
+  Input text  id=loginform-password  ${USERS.users['${username}'].password}
+  Click Element  name=login-button
+
+###############################################################################################################
+######################################    СТВОРЕННЯ ТЕНДЕРУ    ################################################
+###############################################################################################################
+
+Створити тендер
+  [Arguments]  ${username}  ${tender_data}
+  ${items}=  Get From Dictionary  ${tender_data.data}  items
+  ${number_of_items}=  Get Length  ${items}
+  ${tenderAttempts}=   Convert To String   ${tender_data.data.tenderAttempts}
+  run keyword and ignore error  log  ${tender_data.data.items}
+  run keyword and ignore error  log  ${tender_data.data.items[0]}
+  run keyword and ignore error  log  ${tender_data.data.items[0]}
```

Run Keyword And Ignore Error

---

і для чого двічі run keyword and ignore error  log  ${tender_data.data.items[0]}?